### PR TITLE
PLAT-66: Remove the setTimeout function from App.js file

### PR DESCRIPTION
**Pull request by dondrzzy**
## JIRA TICKET NAME
[PLAT-66: Remove the setTimeout function from App.js file](https://issues.openmrs.org/browse/PLAT-66)

## Description of what I changed

- Removed the setTimeout function from App.js file

## Issue I worked on
[PLAT-66](https://issues.openmrs.org/browse/PLAT-66)

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [ ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: app/js/components/App.jsx [hunk 1]
```diff
@@ -49,9 +49,7 @@ export default class App extends Component {
   handleError = (error) => {
     if (error.response) {
       this.toastError(error.response.data.error.message);
-      setTimeout(() => {
-        this.props.history.push('/login');
-      }, 3000);
+      this.props.history.push('/login');
```
Comment (dotNesh) [inline app/js/components/App.jsx:52]: Why is the timeout being removed and why was it there before?

**Review by muhozi — APPROVED** [review]
LGTM

**Review by emmabaye — APPROVED** [review]
LGTM

File: app/js/components/App.jsx (hunk 1 above)
Comment (dondrzzy) [reply app/js/components/App.jsx:52]: @kmunene, the toastr at the time was in the same component and if I redirected immediately then the notification wouldn't appear. However, I created a ticket to take the toastr out of the Router scope so the setTimeout is no longer necessary.

Comment (dotNesh) [reply app/js/components/App.jsx:52]: Okay cool.

**Review by dotNesh — APPROVED** [review]

**Review by Daramola98 — APPROVED** [review]
LGTM

**Pull request merged**